AutoMapper.Data/AutoMapper.Data.csproj

```diff
@@ -25,7 +25,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AutoMapper" Version="[12.0.0, 13.0.0)" />
+    <PackageReference Include="AutoMapper" Version="[13.0.1,)" />
```

Comment:
The version range must be 13-14 and it must target .net 6.

Reply:
Updated to target .net 6 and the version range for the AutoMapper package